### PR TITLE
Updating changelog after triggered build

### PR DESCRIPTION
**Pull request by maxzieba**
## Proposed changes

Updating changelog after triggered build

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [X] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [X] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Review by jeferal — APPROVED** [review]

File: docs/user_guide/5_changelog.md [hunk 1]
```diff
@@ -3,6 +3,38 @@
...
+* Removing deprecated field from general_info
+* Fix phantom hand
+* Removed old launch file with box and replaced with the new one from sr_interface
+* support ImageMagick 6 and 7
+* hand side fix error
+
+### Version 1.0.3
+* Migrating to dae and adding materials
+* Fixing the color of wrist mesh
+* Switching to new xacros
```
Comment (beatrizleon) [inline docs/user_guide/5_changelog.md:28]: Maybe delete those two. it doesn't look great to put the "revert" one

File: docs/user_guide/5_changelog.md [hunk 2]
```diff
@@ -3,6 +3,38 @@
 
 ## ROS Noetic
 
+### Version 1.0.4
+
+* Xacros refactored
+* removing deprecated sr hand package
```
Comment (beatrizleon) [inline docs/user_guide/5_changelog.md]: Can you please change all to start with a capital letter?

File: docs/user_guide/5_changelog.md [hunk 3]
```diff
@@ -3,6 +3,38 @@
...
+### Version 1.0.3
+* Migrating to dae and adding materials
+* Fixing the color of wrist mesh
+* Switching to new xacros
+* Revert "Switching to new xacros"
+* Adding piezofeedback driver
+* Update args
+* Adding a way of exiting the demo
+* Edit tactile threshold
+* Adding New Hand
```
Comment (beatrizleon) [inline docs/user_guide/5_changelog.md]: What is this one?

File: docs/user_guide/5_changelog.md [hunk 4]
```diff
@@ -3,6 +3,38 @@
...
+* support ImageMagick 6 and 7
+* hand side fix error
+
+### Version 1.0.3
+* Migrating to dae and adding materials
+* Fixing the color of wrist mesh
+* Switching to new xacros
+* Revert "Switching to new xacros"
+* Adding piezofeedback driver
+* Update args
```
Comment (beatrizleon) [inline docs/user_guide/5_changelog.md]: What is this one?

File: docs/user_guide/5_changelog.md [hunk 5]
```diff
@@ -3,6 +3,38 @@
...
+* Removed old launch file with box and replaced with the new one from sr_interface
+* support ImageMagick 6 and 7
+* hand side fix error
+
+### Version 1.0.3
+* Migrating to dae and adding materials
+* Fixing the color of wrist mesh
+* Switching to new xacros
+* Revert "Switching to new xacros"
+* Adding piezofeedback driver
```
Comment (beatrizleon) [inline docs/user_guide/5_changelog.md]: is this part of the hand image?

**Review by beatrizleon — CHANGES_REQUESTED** [review]

File: docs/user_guide/5_changelog.md (hunk 5 above)
Comment (maxzieba) [reply docs/user_guide/5_changelog.md]: It was a left over from moving the piezofeedback driver from common_resources to another repo. Removed the line and reverting

File: docs/user_guide/5_changelog.md (hunk 4 above)
Comment (maxzieba) [reply docs/user_guide/5_changelog.md]: Update args is regarding this https://github.com/shadow-robot/sr_interface/pull/718/files. Updated the text.

File: docs/user_guide/5_changelog.md (hunk 3 above)
Comment (maxzieba) [reply docs/user_guide/5_changelog.md]: I talked to Fotios to exclude PR's towards sr_hand_config as it only contains new calibrations so not relevant to the changelog. Removed the entry here (missed this, but I remove many others like this)

File: docs/user_guide/5_changelog.md (hunk 2 above)
Comment (maxzieba) [reply docs/user_guide/5_changelog.md]: Done

File: docs/user_guide/5_changelog.md (hunk 1 above)
Comment (maxzieba) [reply docs/user_guide/5_changelog.md:28]: OK

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**